README.md

```diff
@@ -110,17 +110,16 @@ This plugin defines hooks for `environment`, `checkout`, `command`, and `post-co
 
 #### Available properties
 
-- `input_artifacts`: String array of artifact paths to download and sign.
+- `input_artifacts`: String array of artifact paths to download that are required to sign.
```

Comment:
Unsure - since we could potentially be required to download a folder, I left it as a list for flexibility. If we can come to an agreement regarding a nice syntax to unify input/output (eg, is adding a `/*` on download and `/**/*` if a target ends with `.app` reliable enough?), then I would be more than fine with making it a simple string.

Reply:
I've updated the plugin to use a single input artifact, treating `.app` and `.framework` as directories